virttest/utils_misc.py

```diff
@@ -1562,6 +1562,54 @@ def add_identities_into_ssh_agent():
     process.run("ssh-add")
 
 
+def gen_lvmap_mpath(session=None):
+    """
+    Return the name mapping between logical volume and device
+    mapper multipathed device.
+
+    :param session: Guest session
```

Comment:
may update doc string as below:
:param session: default is None, it means it access host,otherwise Guest Vm session

Reply:
doc string updated